src/resizable.directive.ts

```diff
@@ -754,7 +761,10 @@ export class ResizableDirective implements OnInit, OnChanges, OnDestroy {
     this.mouseup.subscribe(() => {
       if (currentResize) {
         this.renderer.removeClass(this.elm.nativeElement, RESIZE_ACTIVE_CLASS);
-        this.renderer.setStyle(document.body, 'cursor', '');
+        // browser check for angular universal, because it doesn't know what document is
+        if (isPlatformBrowser(this.platformId)) {
+          this.renderer.setStyle(document.body, 'cursor', '');
```

Comment:
Same here, I'm not sure this check is needed here as a mouse up event can never fire on the server